src/detect-lua.c

```diff
@@ -1413,12 +1411,12 @@ static int LuaMatchTest01a(void)
     int r = AppLayerParserParse(
             NULL, alp_tctx, &f, ALPROTO_HTTP1, STREAM_TOSERVER, httpbuf1, httplen1);
     if (r != 0) {
-        printf("toserver chunk 1 returned %" PRId32 ", expected 0: ", r);
+        printf("toserver chunk 1 returned %" PRId32 ", expected 0: ", r); //
         goto end;
     }
```

Comment:
You may just convert this to the FAIL/PASS API, this would make the `printf` statement obsolete, because implicit.